src/components/Dropdown.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useEffect, useCallback } from "react";
+import SVGIcon from "@/src/components/SVGIcon";
+import {
+  DropdownContainer,
+  DropdownLabel,
+  DropdownButton,
+  DropdownMenu,
+  DropdownItem,
+  ErrorMessage,
+  IconWrapper,
+} from "@/src/components/styles/Dropdown";
+
+const Dropdown = ({
```

Comment:
This is not the way we learned to code these. It would be nice for the review to handle it like we are used to it. :)

Reply:
but it works, and yes I change it as we learnt it